lib/util/hud_lists.rb

```diff
@@ -0,0 +1,1611 @@
+      3 => '6-10',
+      4 => '11-20',
+      5 => 'More than 20',
+      8 => "Client doesn't know",
+      9 => 'Client refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def data_collection_stage_map
+    {
+      1 => 'Project entry',
+      2 => 'Update',
+      3 => 'Project exit',
+      5 => 'Annual assessment',
+      6 => 'Post-exit',
+    }.freeze
+  end
+
+  def ad_hoc_yes_no_map
+    {
+      0 => 'No',
+      1 => 'Yes',
+      8 => "Don't Know",
+      9 => 'Refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def wellbeing_agreement_map
+    {
+      0 => 'Strongly disagree',
+      1 => 'Somewhat disagree',
+      2 => 'Neither agree nor disagree',
+      3 => 'Somewhat agree',
+      4 => 'Strongly agree',
+      8 => "Client doesn't know",
+      9 => 'Client refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def feeling_frequency_map
+    {
+      0 => 'Not at all',
+      1 => 'Once a month',
+      2 => 'Several times a month',
+      3 => 'Several times a week',
+      4 => 'At least every day',
+      8 => "Client doesn't know",
+      9 => 'Client refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def moving_on_assistance_map
+    {
+      1 => 'Subsidized housing application assistance',
+      2 => 'Financial assistance for Moving On (e.g., security deposit, moving expenses)',
+      3 => 'Non-financial assistance for Moving On (e.g., housing navigation, transition support)',
+      4 => 'Housing referral/placement',
+      5 => 'Other (please specify)',
+    }.freeze
+  end
+
+  def current_school_attended_map
+    {
+      0 => 'Not currently enrolled in any school or educational course',
+      1 => 'Currently enrolled but NOT attending regularly (when school or the course is in session)',
+      2 => 'Currently enrolled and attending regularly (when school or the course is in session)',
+      8 => "Client doesn't know",
+      9 => 'Client refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def most_recent_ed_status_map
+    {
+      0 => 'K12: Graduated from high school',
+      1 => 'K12: Obtained GED',
+      2 => 'K12: Dropped out',
+      3 => 'K12: Suspended',
+      4 => 'K12: Expelled',
+      5 => 'Higher education: Pursuing a credential but not currently attending',
+      6 => 'Higher education: Dropped out',
+      7 => 'Higher education: Obtained a credential/degree',
+      8 => "Client doesn't know",
+      9 => 'Client refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def current_ed_status_map
+    {
+      0 => 'Pursuing a high school diploma or GED',
+      1 => "Pursuing Associate's Degree",
+      2 => "Pursuing Bachelor's Degree",
+      3 => 'Pursuing Graduate Degree',
+      4 => 'Pursuing other post-secondary credential',
+      8 => "Client doesn't know",
+      9 => 'Client refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  def hud_code_to_function_map
```

Comment:
@eanders what do you think of this file? I think we should keep the code-generated file as barebones as possible. This should be all we need to write the translation and lookup functions by hand in the `HUD` module.

Reply:
agreed!